tracee/tracee.go

```diff
@@ -638,19 +638,19 @@ func (t *Tracee) populateBPFMaps() error {
 	if err != nil {
 		return fmt.Errorf("error getting BPF program trace_ret_vfs_write_tail: %v", err)
 	}
-	bpfProgArrayMap.Update(uint32(tailVfsWrite), uint32(prog.GetFd()))
```

Comment:
can you please explain this suggestion? `tailVfsWrite` is a uint32 anyway

Reply:
Yes exactly - it's an uint32 already so this cast is unnecessary.

Reply:
oh right, I read it the other way

Reply:
read this the wrong way :) thanks